main/postgresql/APKBUILD

```diff
@@ -119,6 +122,46 @@ contrib() {
 	rmdir -p "$subpkgdir"/usr/share/doc/postgresql || true
 }
 
+plperl() {
+	depends="postgresql perl"
```

Comment:
I will remove the hardcoded depends. [DONE]

To my knowledge it can only be build against 1 version, I will start research to see If both are possible, If not I will switch it to Python3.

I concur with you about the ```-contrib```, the question that remains is, either a ```postgresql-pl-contrib``` or ```postgresql-plperl-contrib``` etc.. WDYT ?